q2_sample_classifier/plugin_setup.py

```diff
@@ -639,6 +640,22 @@
                 'displays (normalized) feature abundances per sample.'
 )
 
+plugin.methods.register_function(
+    function=shapely_values,
+    inputs={**inputs, 'sample_estimator': SampleEstimator[Classifier]},
+    parameters={},
+    outputs=[('shap', SampleData[Probabilities])],
```

Comment:
Is there a particular reason for choosing this QIIME2 type over `FeatureData[Importance]` here?

Reply:
When we first wrote this, it wasn't clear if the dimensions would work out with `FeatureData[Importance]` -- Shapely values are of dimensions `D x N` for `D` features and `N` samples.

Reply:
understood 👍🏼 I think in this case it could make sense to add a new type specifically for shapely values (as they are distinct from predicted sample probabilities and feature importances).